src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -805,14 +805,20 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         }
 
         /// <summary>
-        /// Create a <see cref="LinearSvmTrainer"/>, which predicts a target using a linear binary classification model trained
```

Comment:
>a  [](start = 19, length = 2)

let's keep this

Reply:
That will make it inconsistent with rest of trainer documentation.

---
In reply to: [278372222](https://github.com/dotnet/machinelearning/pull/3575#discussion_r278372222) [](ancestors = 278372222)